gobblin-runtime/src/main/java/org/apache/gobblin/scheduler/JobScheduler.java

```diff
@@ -396,7 +396,7 @@ public void scheduleJob(Properties jobProps, JobListener jobListener, Map<String
     try {
       // Schedule the Quartz job with a trigger built from the job configuration
       Trigger trigger = createTriggerForJob(job.getKey(), jobProps, Optional.absent());
-      this.scheduler.getScheduler().scheduleJob(job, trigger);
+      this.scheduler.getScheduler().scheduleJob(job, trigger); //quartz scheduler; calls runJob()
```

Comment:
not sure this comment needs to stay... if so, can we fold into one on 397?

Reply:
removed the comment